workspaces/quay/plugins/quay/app-config.yaml

```diff
@@ -0,0 +1,12 @@
+dynamicPlugins:
```

Comment:
Please rename this file to `app-config.dynamic.yaml` so that's clear that it's related t dynamic plugins and that it's not a common app-config.